app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function ($filter,
+    $q,
+    $uibModal,
+    $rootScope,
+    APIService,
+    DataService,
+    Logger,
+    NotificationsService) {
+
+    var mobileclientVersion = {
+      group: "mobile.k8s.io",
+      version: "v1alpha1",
+      resource: "mobileclients"
+    };
+    var remove = function (apiObject) {
```

Comment:
I would put "delete" in the function name to make it clear the resource is really being deleted from the API server. `remove` doesn't convey that to me.